<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatonusTestLoader</class>
 <widget class="QMainWindow" name="PlatonusTestLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Platonus Test Loader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDateEdit" name="startDateEdit">
        <property name="displayFormat">
         <string>d/M/yyyy</string>
        </property>
        <property name="date">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>20</day>
         </date>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="finishDateEdit">
        <property name="displayFormat">
         <string>d/M/yyyy</string>
        </property>
        <property name="date">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>13</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>380</width>
         <height>455</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="testingNamesLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logOutButton">
      <property name="text">
       <string>Log &amp;Out</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="downloadAllCheckBox">
      <property name="text">
       <string>Download &amp;All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
